firefox_ui_tests/remote/security/test_untrusted_connection_error_page.py

```diff
@@ -29,4 +29,8 @@ def test_untrusted_connection_error_page(self):
 
         button = self.marionette.find_element(By.ID, "getMeOutOfHereButton")
         button.click()
-        Wait(self.marionette).until(lambda mn: target_url == self.marionette.get_url())
+        Wait(self.marionette).until(
+          lambda mn: (
+            target_url == self.marionette.get_url()
+          )
```

Comment:
The PEP8 error on the build failure indicated a line limit of 99 characters, and I reproduced this error locally.